lib/stdio/utils.js

```diff
@@ -7,3 +8,16 @@ export const isBinary = value => isUint8Array(value) || Buffer.isBuffer(value);
 
 const textDecoder = new TextDecoder();
 export const binaryToString = uint8ArrayOrBuffer => textDecoder.decode(uint8ArrayOrBuffer);
+
+// Like `source.pipe(destination)`, if `source` ends, `destination` ends.
+// Like `stream.pipeline(source, destination)`, if `source` aborts/errors, `destination` aborts.
+// Unlike `stream.pipeline(source, destination)`, `destination` behavior does not propagate to `source`.
+export const pipeline = async (source, destination) => {
```

Comment:
I would call it something else so it's not confused at callsite with `Stream.pipeline`

Reply:
Fixed in 1533da2